new_01-vim-beginner.tutor

```diff
@@ -0,0 +1,1234 @@
+Try it out.
+
+As you have seen the «n» and «N» keys are now reversed as well. It's easier to
+issue one backwards search and then keep hitting «n» than hitting «N»
+repeatedly.
+
+The pattern does not have to be just a word, it can be any regular expression.
+Try searching for "f.x". You will find matches for both "fox" and "fix".
+
+
+# Summary
+
+asdf
+
+
+# Part 3: VISUAL MODE
+
+Visual mode is similar to selecting text in other text editors. We define a
+visual selection over a piece of text and execute and action or command over
+that selection.
+
+# Lesson 3.1: VISUAL MODE
+
+** Press «v» to select a sequence of text **
+
+  1. Move the cursor on first character of the first word to change.
+
+  2. Press «v» to enter visual mode.
+
+  3. Use the motions you know to move the selection to the last character of
+     the last word to change.
+
+  4. Press «c» to change the selected portion of text.
+
+---> Vim is a horribly complicated to use text editor. {expect:Vim is a very efficient and productive text editor.}
+---> Vim is a very efficient and productive text editor. {expect:Vim is a very efficient and productive text editor.}
+
+We want to change as much text as possible, and counting words is slow. Visual
+mode allows us to make an exact selection before applying our action.
+
+You can use counts on the motions when defining your selection, but when
+performing the operation we cannot. There is no reason anyway, the selection
+takes the place of count and motion.
+
+Visual mode is very nice, but don't overuse it. Take the following example:
+
+---> Vim's usability is an utter and complete mess conceived by madmen. {expect:Vim's usability allows users to edit text as fast as they think.}
+---> Vim's usability allows users to edit text as fast as they think. {expect:Vim's usability allows users to edit text as fast as they think.}
+
+It is faster to position the cursor on the first wrong word and then change
+'til the period using «c»«t»«.».
+
+# Lesson 3.2: VISUAL LINE MODE
+
+** Select entire lines with «V» **
+
+There are two pairs of lines wrong. We don't want to fix them one at a time, so
+we'll transplant two lines at a time.
+
+  1. Move the cursor on the first line.
+
+  2. Press «V» to select the entire line. Move to cursor down.
+
+  3. Press «d» to delete the selection, move to cursor on the second line.
+
+  4. Press «p» to put the two lines below the cursor.
+
+---> c) Intelligence is learned, {expect:NULL}
+---> d) Can you learn too? {expect:NULL}
+---> a) Roses are red, {expect:NULL}
+---> b) Violets are blue, {expect:NULL}
+
+We could have used «2dd» here as well, but in practice we will often have to
+transplant entire paragraphs consisting of tens of lines.
+
+# Lesson 3.3: VISUAL BLOCK MODE
+
+** We can also select a rectangular slice of text. **
+
+  1. We want to change the prefix all these items. Move the cursor to the first
+     '\'.
+
+  2. Press <C-V> to enter visual mode.
```

Comment:
Not sure if you wrote this, but IMO this should be `<C-v>`, since while the case of `v` doesn't matter, we don't want users thinking they need to press shift as well.